irods/files/rule-bases/cyverse_logic.re

```diff
@@ -508,21 +509,23 @@ _cyverse_logic_ensureAVUEditable(*EditorName, *EditorZone, *Attr, *Val, *Unit) {
 	}
 }
 
-# If an AVU is not protected, it sets the AVU to the given item
-_cyverse_logic_setAVUIfUnprotected(*EntityType, *EntityName, *Attr, *Val, *Unit) {
-	if (!_cyverse_logic_isAVUProtected(*Attr)) {
+# If an AVU is allowed, it sets the AVU to the given item
+_cyverse_logic_setAVU(*EntityType, *EntityName, *Attr, *Val, *Unit) {
+	if (!_cyverse_logic_isAVUProtected(*Attr) || (_cyverse_logic_isAVUProtected(*Attr) && !_cyverse_logic_isAVUProtectedUUID(*Attr))) {
```

Comment:
Since ipc_UUID is also a protected AVU, `_cyverse_logic_isAVUProtected(*Attr) && !_cyverse_logic_isAVUProtectedUUID(*Attr)` will always evaluate to `false`. The test needs to check if the account attempting to copy the AVU is a `rodsadmin` type user.